### DISCUSSION TITLE
Response time in switching between light and dark modes

### DISCUSSION THREAD

**Discussion by dense-set** [quarto-dev/quarto-cli#8295] [Q&A] [answered]
### Description

I want to incorporate light and dark modes for a Quarto website. I do as is instructed in [documentation](https://quarto.org/docs/output-formats/html-themes.html#dark-mode). Everything works fine except the response time:

https://github.com/quarto-dev/quarto-cli/assets/102873841/f5e13a49-951f-47e9-94ef-f059b41c85b9

The webpages while in the default theme (dark in the above) load fine. But when the secondary mode (light in the above) is selected, webpages have this strange behaviour: The webpages first seem to load in the default theme, and only after some (short) time, do they actually revert to the correct theme. This is obviously not pleasant.

How to correct this? That is, how to have the seamless webpage navigation in the secondary mode like in the default mode?

**Comment by mcanouil** [comment, marked as answer]
I believe this to be a duplicate of:
- https://github.com/quarto-dev/quarto-cli/discussions/5900
